Application/People/Person/Frontend/FrontendStudentGeneral.php

```diff
@@ -334,18 +337,28 @@ private function getEditStudentGeneralForm(TblPerson $tblPerson = null)
          * Panel: Agreement
          */
         $AgreementPanel = array();
+        $CheckboxList = array();
         if(($tblAgreementCategoryAll = Student::useService()->getStudentAgreementCategoryAll())){
             array_walk($tblAgreementCategoryAll,
-                function (TblStudentAgreementCategory $tblStudentAgreementCategory) use (&$AgreementPanel) {
-                    array_push($AgreementPanel, new Aspect(new Bold($tblStudentAgreementCategory->getName())));
+                function (TblStudentAgreementCategory $tblStudentAgreementCategory) use (&$AgreementPanel, &$CheckboxList) {
                     $tblAgreementTypeAll = Student::useService()->getStudentAgreementTypeAllByCategory($tblStudentAgreementCategory);
+                    // Extra Toggle on Category
+                    $CategoryCheckboxList = array();
+                    array_walk($tblAgreementTypeAll,
+                        function (TblStudentAgreementType $tblStudentAgreementType) use (&$AgreementPanel, &$CategoryCheckboxList,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The first parameter of a multi-line use declaration must be on the line after the opening bracket](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)